pkg/dsdk/connection.go

```diff
@@ -198,6 +264,7 @@ func (c *ApiConnection) do(ctxt context.Context, method, url string, ro *greq.Re
 	gurl.Path = path.Join(gurl.Path, url)
 	reqId := uuid.Must(uuid.NewRandom()).String()
 	sdata, err := json.Marshal(ro.JSON)
+	Log().Debugf("REST call going with following payload, %s", string(sdata))
```

Comment:
We don't want to blindly log every request payload, this should only happen if `sensitive` is false.

Reply:
This is from upstream https://github.com/Datera/go-sdk/blob/master/pkg/dsdk/connection.go#L265

Reply:
True but it still looks like it's in the wrong place.. `sdata` is already being logged further down and it's explicitly overwritten just a few lines below here so that the payload is hidden. I don't think that line should've been committed upstream at all.

Reply:
You are right, I removed the line from both PRs.